test/errors/duplicateEnumElement.1.good

```diff
@@ -0,0 +1,23 @@
+─── error in test/errors/duplicateEnumElement.chpl:5 [MultipleEnumElems] ───
+  Enum 'languagesILike' has multiple elements named 'chapel'.
+      |
+    5 | var chpl = languagesILike.chapel;
```

Comment:
I'm surprised that this is pointing to a line where the enum is used rather than the one where it's defined.

Reply:
I think the reason is that the error is thrown when the enum element is accessed. I can tweak it, however.